client/app/scripts/superdesk-workspace/workspace.js

```diff
@@ -324,6 +324,36 @@
                         superdeskFlags.flags.hideMonitoring = false;
                     }
                 };
+
+                /*
+                 * By using keyboard shortcuts, change the current showed view
+                 *
+                 */
+                var opt = {global: true};
+                keyboardManager.bind('alt+h', function (e) {
```

Comment:
I had a talk with @fritzSF about this this morning, and we agreed to use `alt +` for global shortcuts (like changing views) and `ctrl +` for authoring shortcuts.